starter-code/memory.js

```diff
@@ -31,6 +31,38 @@ var MemoryGame = function() {
     this.selectedCards = [];
     this.pairsClicked = 0;
     this.correctPairs = 0;
+  };
+
+
+
+    MemoryGame.prototype._shuffleCards = function() {
```

Comment:
This is nice. It's a good idea to make a seperate utility function that focuses on making one shuffle move, and then looping on that.